windows.go

```diff
@@ -5,6 +5,7 @@ package cagent
 import (
 	"time"
 
+	"github.com/cloudradar-monitoring/cagent/types"
```

Comment:
external imports come first followed by a new line and then our internal imports

Reply:
done